mavros/include/mavros/mavros_uas.h

```diff
@@ -174,7 +174,7 @@ class UAS {
 	/**
 	 * @brief Store IMU data [ENU]
 	 */
-	void update_attitude_imu_enu(sensor_msgs::Imu::Ptr &imu);
+	void update_attitude_imu_nwu(sensor_msgs::Imu::Ptr &imu);
```

Comment:
all the changes on `mavros_uas` and `uas_data` are not correct cause the orientation frame of reference is local ENU, and not body NWU.